plugins/lua/building-hacks.lua

```diff
@@ -147,13 +147,13 @@ function registerBuilding(args)
     end
     --animations and connections next:
     local gears
+    local frames
 
     local frameLength
     local animate=args.animate
     if not auto_gears then
         gears=args.gears or {}
-        frameLength=1
-        local frames
+        frameLength=1        
```

Comment:
Travis was complaining because you have 8 extra spaces at the end of this line. It's easy to fix on my end, so I'll do that, but just something to keep in mind for next time.

Reply:
Fixing that also made the diff cleaner, incidentally: https://github.com/DFHack/dfhack/commit/61df5e52867f8ada817078853f673a090be1e024